index.html

```diff
@@ -625,6 +625,54 @@ <h3>Instead of...</h3>
       </section>
     </section>
 
+    <section>
+      <h2>Version 2</h2>
+      <p class="note">
+        This representation relies on claims registered in the
+        <a href="https://www.iana.org/assignments/jwt/jwt.xhtml#claims">IANA JSON Web Token Claims Registry</a> whenever possible.
+      </p>
+      <p class="note">
+        Production of this representation does not use <code>credential+ld+json</code> as an input.
+      </p>
+      <section>
+        <h2>Credential Metadata</h2>
+        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the content type <code>verifiable-credential+jwt</code>.</p>
```

Comment:
sorry for bike-shedding but what about "vc+jwt"? 
```suggestion
        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the content type <code>vc+jwt</code>.</p>
```